# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1437058654 14400
#      Thu Jul 16 10:57:34 2015 -0400
# Node ID d2c03beac9553e97a1d1bf431a1ed0a8832990a1
# Parent  e365a34a1c5c1387ea2c377415ac1bd2311babdd
don't attempt to distribute obsolete libinterp/Makefile.in

* libinterp/module.mk (EXTRA_DIST): Remove libinterp/Makefile.in from
the list.

diff --git a/libinterp/module.mk b/libinterp/module.mk
--- a/libinterp/module.mk
+++ b/libinterp/module.mk
@@ -77,17 +77,16 @@ LIBINTERP_BUILT_NODISTFILES = \
   libinterp/oct-conf.h \
   libinterp/oct-conf-features.h \
   libinterp/version.h \
   $(OPT_HANDLERS) \
   $(ALL_DEF_FILES) \
   libinterp/builtins.cc
 
 EXTRA_DIST += \
-  libinterp/Makefile.in \
   libinterp/DOCSTRINGS \
   libinterp/config-features.sh \
   libinterp/find-defun-files.sh \
   libinterp/gendoc.pl \
   libinterp/genprops.awk \
   libinterp/mk-errno-list \
   libinterp/mk-pkg-add \
   libinterp/mkbuiltins \
